.travis.yml

```diff
@@ -1 +1,9 @@
 language: crystal
+
+install:
+  - shards install
+
+script:
+  - bin/ameba
+  - crystal tool format --check
+  - crystal spec
```

Comment:
Let's reverse the order - `spec`, `format`, `ameba` - starting from the most important.

Reply:
Have it this way since the first two are more likely to fail, which would fail faster than specs as it gets bigger.  But everything will run so fast i dont think it really matters.

Reply:
That's correct, yet I belive it's more important to firstly check failing specs and only after that tackle formatter/linter related issues.